### ISSUE TITLE
Processing failure with obfuscated filename

### ISSUE THREAD

**Issue by sukhysall** [clinton-hall/nzbToMedia#1405]
Hi

NZBGet: v20.0
Synology DS1511+ (DSM 6)

So the error I'm getting appears to occur with filenames which are obfuscated. My nzbget/Nzbdrone setup has been running fine until very recently although I can't say when exactly. 

My Nzbdrone is configured to deleted failed downloads so heaven knows how many legitimate downloads were pulled down only to be marked as PP-Failure and subsequently deleted because they were classed as failed downloads. 

Is there any way for the PP scripts to change obfuscated filenames to match their folder names? My folder names appear to be fine but it's just the obfuscated filename which is causing me an issue.

I tend not to monitor my downloads much as I have pushover notifications to notify me when something is processed by NzbDrone. So I don't know if EVERY obfuscated filename is being marked as PP-Failed or if the issue has just started recently. 

I've attached a log: 
[Nightflyers.S01E10.720p.WEBRip.x264-TBS English.log](https://github.com/clinton-hall/nzbToMedia/files/2680120/Nightflyers.S01E10.720p.WEBRip.x264-TBS.English.log)

**Comment by clinton-hall** [comment]
These logs show the error is with loading the autoProcessMedia.cfg file and not with the file names. According to this log I would expect ALL downloads to fail.

Try deleting the autoProcessMedia.cfg file and see if it runs correctly.

**Comment by sukhysall** [comment]
Update:

So I puttied into the nzbtomedia scripts folder and I saw 3 autoProcessMedia.cfg files (the other 2 were .old and .spec respectively).

I deleted the first autoProcessMedia.cfg file and left the other 2 files and I re-ran the PP and it was successful.

HOWEVER, upon performing a PP for another freshly downloaded episode, the PP failed. I then puttied back into the same folder and saw that the file I'd deleted was now back.

I then performed the same action of deleting the first file and re-performed the PP and again it was successful. 

I have now started nzbget as "root" and throughout the entire nzbget folder I have set everything as "root" and I'm still seeing the same issue.

Help?

**Comment by labrys** [comment]
`autoProcessMedia.cfg` should be your active configuration.  `.old` is a backup and `.spec` is the default unconfigured configuration.  Check your configuration.

**Comment by clinton-hall** [comment]
it sounds like something is being written into your config (from NZBGet) that writes, but fails to read second time.

Can you email your autoProcessMedia.cfg file to `fock_wulf@hotmail.com`?
Edit out your api keys etc first.
Hopefully I can figure out where this fails and advise what needs to be changed in your NZBGet settings.

**Comment by sukhysall** [comment]
Hiya

I've resolved the issue as per below:

So I replaced the autoprocessMedia.cfg  with the ".spec" copy. After an NzbGet reload, I could still see the information I'd already populated in the relevant PP fields. I assume this was because of the nzbget.conf file storing my config and that turned out to be correct. A quick backup of the nzbget.conf and checking it in Notepad++ confirmed my PP parameters were kept by Nzbget.

I had originally assumed if I deleted the autoProcessMedia.cfg that the parameters would disappear. Didn't look like that was the case.

In the end I just did a complete wipe of Nzbget and started from scratch. I manually repopulated each Nzbget parameter and each PP parameter to what they were set as prior and it seems to have done the trick.

I've had comics, tv shows and books processing correctly.
